core/src/main/kotlin/gg/destiny/lizard/core/chat/ChatSocket.kt

```diff
@@ -125,6 +125,9 @@ class ChatSocket(
     val type = Message.of(message.substringBefore(" ", UNKNOWN_TYPE))
     val data = message.substringAfter(' ')
     L { "Message: $message" }
+    if (message.substringBefore(" ") == "ERR") {

```

Comment:
We already deserialize err messages properly so that we can handle them, so lets remove this.